frontend/src/views/settings.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { Redirect } from '@reach/router';
+
+import { UserTopBar } from '../components/user/settings';
+import { OSMCard } from '../components/user/content';
+import { UserInformationForm, UserInterests } from '../components/user/forms';
+
+
+export function Settings() {
+  const userIsloggedIn = useSelector(state => state.auth.get('token'));
+  const userDetails = useSelector(state => state.auth.get('userDetails'));
+
+  if (userIsloggedIn) {
+    return (
+      <div className="pull-center">
+        <UserTopBar />
```

Comment:
If it is possible the user sees a page like this immediately after logging in, I think maybe we should wrap things like these in a loading component like `<React-placeholder ready={redux-says-it-is-loaded}>` so it doesn't flash from "Mapper" to your user name and then flash again to add the changeset count. Maybe also the input fields in personal information: on slow connections they could lose their work users start typing.
![settings-animation](https://user-images.githubusercontent.com/283343/63474375-fc3b2480-c446-11e9-926e-47ee4139f314.gif)
When you refresh, sometimes the user image even shows as a strange shape while it loads.

(also I clearly need to map with smaller changesets so I can get to a higher level 😊 than beginner )

Reply:
I added the placeholders. On the form, I decided to keep it disabled until the data is loading.

Reply:
I solved the image problem by setting the width.